static/src/js/components/PublishPreview/PublishPreview.jsx

```diff
@@ -274,264 +191,186 @@ const PublishPreview = ({ isRevision }) => {
-                    pill
-                  >
-                    { tagCount }
-                  </Badge>
-                </Button>
-                <Button
-                  className="btn btn-link"
-                  type="button"
-                  variant="link"
-                  disabled
-                >
-                  Granules
-                  <Badge
-                    className="m-1"
-                    bg="secondary"
-                    pill
-                  >
-                    { granuleCount }
-                  </Badge>
-                </Button>
-
-                <Button
-                  className="btn btn-link"
-                  type="button"
-                  variant="link"
-                  onClick={
-                    () => {
-                      handleCreateAssociatedVariable()
-                    }
-                  }
-                >
-                  Create Associated Variable
-                </Button>
-              </>
-            )
-          }
+            icon: FaDownload,
+            onClick: handleDownload,
+            title: 'Download JSON'
+          },
+          // {
+          //   icon: FaPlus,
+          //   onClick: handleCreateAssociatedVariable,
+          //   title: 'Create Associated Variable'
+          // },
           {
-            derivedConceptType !== conceptTypes.Collection && (
-              <Button
-                className="btn btn-link"
-                type="button"
-                variant="link"
-                onClick={
-                  () => {
-                    handleManageCollectionAssociation()
-                  }
-                }
-              >
-                Manage Collection Association
-              </Button>
-            )
-          }
-          <Button
-            className="btn btn-link"
-            type="button"
-            variant="link"
-            onClick={handleRevisions}
-          >
-            Revisions
-            <Badge
-              className="m-1"
-              bg="secondary"
-              pill
-            >
-              { revisionCount }
-            </Badge>
-          </Button>
-          <Button
-            type="button"
-            variant="outline-danger"
-            onClick={
-              () => {
-                toggleShowDeleteModal(true)
-              }
-            }
-          >
-            Delete
-            {' '}
-            {derivedConceptType}
-            {' '}
-            Record
-          </Button>
-          <CustomModal
-            message="Are you sure you want to delete this record?"
-            show={showDeleteModal}
-            size="lg"
-            toggleModal={toggleShowDeleteModal}
-            actions={
-              [
+            icon: FaEye,
+            to: `/${pluralize(derivedConceptType).toLowerCase()}/${conceptId}/revisions`,
+            title: 'View Revisions',
+            count: revisionCount
+          },
+          ...(
```

Comment:
I agree with the overall sentiment here, I don't think its hard to understand for me, but I think that any mid level dev and lower will be very thrown off by this, and I like the idea of having this be more 'verbose' or pulled out and created in a more clear way. 

That being said, I don't think it would be in scope of this effort.